index.html

```diff
@@ -233,6 +233,59 @@
       </p>
       <section>
         <h3>
+          Installing via automated prompting
+        </h3>
+        <p>
+          User agents that support an <a>automated install prompt</a> MUST run
+          the following steps before initiating an install process:
+        </p>
+        <ol>
+          <li>If a <a>manually</a> instantiated <a>installation process</a> for
+          this web application, terminate this algorithm.
+          </li>
+          <li>Let <var>event</var> be a newly constructed
+          <a>BeforeInstallPromptEvent</a> event named "beforeinstallprompt",
+          which does not bubble and is cancellable.
+          </li>
+          <li>Let <var>prompt promise</var> be the promise corresponding to the
+          event's prompt() method.
+          </li>
+          <li>Let <var>user-choice promise</var> be the promise corresponding
+          to the event's <code>prompt()</code> method.
+          </li>
+          <li>Fire <var>event</var> at the <code>Window</code> object.
+          </li>
+          <li>Queue a task on the Realm’s settings object’s responsible event
```

Comment:
Realm is basically the global object: 
https://tc39.github.io/ecma262/#sec-code-realms